oci/layout/oci_dest_test.go

```diff
@@ -27,7 +28,7 @@ func TestPutBlobDigestFailure(t *testing.T) {
 	defer os.RemoveAll(tmpDir)
 	dirRef, ok := ref.(ociReference)
 	require.True(t, ok)
-	blobPath, err := dirRef.blobPath(blobDigest)
+	blobPath, err := dirRef.blobPath(blobDigest, "")
```

Comment:
Would it be possible to create a test that uses a sharedblobdir?

Reply:
it's here https://github.com/containers/image/pull/369/files#diff-b9397fa64a169d9bf8d84edd77f3b245R291 no?

Reply:
Oh Jeez Louise, ignore me.  Spot on @runcom, I don't know how I managed to miss that.  Obviously time to dive into my first cup of tea for the day.  
Straight up LGTM.